frontend/uddhar-frontend/src/coordinator/components/EmergencyMsg.jsx

```
@@ -13,7 +13,7 @@ const EmergencyMsg = () => {
           ></textarea>
         </div>
         <div className="flex justify-center">
-          <button className="bg-amber-300 text-black px-5 py-2 rounded-md my-2 hover:bg-amber-400">
+          <button className="bg-amber-300 text-black px-5 py-2 rounded-md my-2 hover:bg-amber-400 cursor-pointer">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)